src/probeinterface/probe.py

```diff
@@ -1067,7 +1100,7 @@ def to_zarr(self, folder_path: str | Path) -> None:
         self.add_probe_to_zarr_group(zarr_group)
 
     @staticmethod
-    def from_zarr_group(group: zarr.Group) -> "Probe":
+    def from_zarr_group(group: "zarr.Group") -> "Probe":
```

Comment:
How did this type-hint not break things? I didn't see where zarr was imported?

Reply:
Because we are not testing this properly I guess.

Reply:
I thought it normally failed at import but maybe it is different for staticmethod decorator? I don't know. But nice catch!

Reply:
Yeah, I don't know what's going on.